vdirsyncer/cli/utils.py

```diff
@@ -174,10 +181,15 @@ def get_status_path(base_path, pair, collection=None, data_type=None):
     return path
 
 
-def load_status(base_path, pair, collection=None, data_type=None):
+def load_status(
+    base_path: str,
+    pair: str,
+    collection: str | None = None,
+    data_type: str | None = None,
+) -> dict[str, Any] | None:
```

Comment:
With the changes made below, this function can no longer return `None`.

Reply:
I think it's a mistake I made while rebasing some ruff fixes. I.ll fix all that.